tests/phpunit/src/BltProjectTestBase.php

```diff
@@ -208,6 +208,7 @@ protected function importDbFromFixture($root = NULL, $uri = 'default') {
     $drush_bin = $this->sandboxInstance . '/vendor/bin/drush';
     $this->execute("$drush_bin sql-drop --root=$root --uri=$uri", NULL, FALSE);
     $this->blt('drupal:hash-salt:init');
+    $this->blt('drupal:deployment-identifier:init');
```

Comment:
Why do we do this here? Is it necessary outside of a deployment? If so, should we generate one for all users after initial setup, rather that only in the test case?

Reply:
I believe it is, necessary. @danepowell, can you confirm? As mentioned in an earlier comment, we were previously creating this as part of the [`drupal:install` command](https://github.com/acquia/blt/blob/ceb6e8e3becf5338a7709d2d528a0fb7128729cc/src/Robo/Commands/Setup/BuildCommand.php#L40) (in addition to as part of putting together the build artifact).

Also, this PR does generate one for all users as part of setup. It stops invoking it from the `drupal:install` command and adds it as a command to invoke [as part of `setup:all`](https://github.com/acquia/blt/pull/2726/files#diff-2214b274c5704bd37ecb1d740ac17a20R29).

Reply:
The deployment identifier is most important for deployments (duh), but it's also useful for preventing unexpected behavior in local environments due to stale twig/apc caches. See discussion here: https://github.com/acquia/blt/issues/1568